src/formInput/FormInput.tsx

```diff
@@ -136,6 +132,14 @@ type FormInputProps = {
    * if a variant floating is specified it will add a class 'dcx-floating-label' for supporting a floating label feature
    */
   variant?: 'floating' | 'floating-filled' | 'normal';
+  /**
+   * visually hidden text for screen readers
+   */
+  hiddenErrorText?: string;
```

Comment:
please make it compulsory
```suggestion
  hiddenErrorText: string;
```

Reply:
I do agree, it is a change that will need consumers to add this new prop even if they do not what to use it.

Reply:
it's a major change we doing... and having one breaking change small like this one Is not the end. On the other side making this field mandatory helps all the applications to become more accessible and it will be more dangerous having apps not accessible than adding a simple field to your apps.